app/views/layouts/_redirect.html.erb

```diff
@@ -0,0 +1,34 @@
+<div class="modal fade" id="redirect" tabindex="-1" role="dialog" aria-hidden="true">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-header">
+        <p class="modal-title text-center title-font" id="gridSystemModalLabel"><%= t :explain_redirect %></p>
```

Comment:
`gridSystemModalLabel` is twice, so it is not a valid id :stuck_out_tongue: 

Moreover, I think such Id is already in other html element. I think I copy-pasted from some example in the internet. My bad :cry: